stream_alert/rule_processor/classifier.py

```diff
@@ -252,10 +258,12 @@ def _parse(self, payload):
         if not schema_matches:
             return False
 
+        LOGGER.debug('Schema Matched Records: \n%s', json.dumps([schema_match.parsed_data for schema_match in schema_matches], indent=4))
```

Comment:
Break this only multiple lines - please pylint/autopep or both. Also indent=2 again